Device.js

```diff
@@ -1,4 +1,5 @@
-var { NativeModules, Dimensions } = require('react-native');
+var { NativeModules, NativeAppEventEmitter, Dimensions } = require('react-native');
```

Comment:
Line is too long.<br>'destructuring expression' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.